service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FileUploadNotificationReceiver.java

```diff
@@ -21,33 +21,6 @@
     private final ExecutorService executor = Executors.newFixedThreadPool(3);
     private final AmqpFileUploadNotificationReceive amqpFileUploadNotificationReceive;
 
-    /**
-     * Constructor to verify initialization parameters
-     * Create instance of AmqpReceive
-     * @param hostName The iot hub host name
-     * @param userName The iot hub user name
-     * @param sasToken The iot hub SAS token for the given device
-     * @param iotHubServiceClientProtocol The iot hub protocol name
-     */
-    FileUploadNotificationReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol)
```

Comment:
this and below - package private constructors that are not referenced anywhere in our project, so I removed them.